src/StatusTable/statusTableStyles.ts

```diff
@@ -0,0 +1,176 @@
+import styled from '@emotion/styled';
+
+export const StatusTableStyle = styled.div<any>(({ StatusTableCard, theme }) => ({
```

Comment:
Is `StatusTableCard` just a boolean? If so, make it camelCase so we don't confuse with with a React component. (if it is a component, ignore my comment). Also, instead of `any` here, create an interface above:

```jsx
interface StatusTableStyleProps {
  statusTableCard?: boolean;
}

export const StatusTableStyle = styled.div<any>(({ statusTableCard, theme }) => ({
```